### PR TITLE
Ch 7 review

### PR DESCRIPTION
**Pull request by mariaruth**
### Preparing written documents for publication
- [x] I softened the recommendation for Overleaf, since we don't necessarily strongly recommend it for DIME teams. Would be good to briefly discuss
- [x] i think the note that we used LaTeX/Git to prepare this book would be better in Chapter 7 than where it currently is in Chapter 6 (since that dynamic docs section focuses on intermediate outputs) @bbdaniels 
- [x] i found the usage of output to mean variously tables/figures/research papers to be confusing. i made revisions to try to clarify. - Luiza used code output vs. research output

### Preparing and publishing data
- [x] de-identifying data for publication subsection: there isn't a very clear flow between paragraphs in this subsection. i made substantial revisions.  Also, the first paragraph of the next subsection (publishing raw and constructive datasets) was repetitive so I cut from there and combined with the last paragraph of de-idnetification. it would be good for someone else to read through and make sure that the logical flow of this sub-section is now clear. 

### Preparing a complete reproducibility package
- [x] add link to our reproducibility checklist
- [x] add link to DIME standards

### General
- I made a lot more revisions to this chapter than others, so it would definitely be good to have someone take another pass
- Need to add a final paragraph tying it together and transitioning to the conclusion

### CODE REVIEW

File: chapters/7-publication.tex [hunk 1]
```diff
@@ -642,8 +605,8 @@ \subsection{Releasing a reproducibility package}
   \url{https://codeocean.com}}
 offer to store both code and data in one repository,
 and also provide an online workspace in which others can execute and modify your code
-without having to download your tools and match your local environment
-when packages and other underlying software may have changed since publication.
+without having to download your tools and match your local environment. 
+This is particularly useful over time, as packages and other underlying software may have changed since publication.
 In a similar spirit, attempts to create ``metadata archives''
 that link all the different materials, releases, and publications
 relating to specific research projects or studies
```
Comment (luizaandrade) [inline chapters/7-publication.tex:612]: This seems very broad, I'm not sure I understand what it is referring to.

File: chapters/7-publication.tex [hunk 2]
```diff
@@ -1,7 +1,7 @@
 %------------------------------------------------
 
 \begin{fullwidth}
-Publication typically involves multiple iterations of manuscript,
+Publication typically involves many iterations of manuscript,
```
Comment (kbjarkefur) [inline chapters/7-publication.tex:4]:
```suggestion
Publication typically involves many iterations of preparing manuscript,
```
I might be wrong, but I feel like this sentence is missing a word.

File: chapters/7-publication.tex [hunk 3]
```diff
@@ -67,38 +66,39 @@ \subsection{Preparing manuscripts as {\LaTeX} documents}
...
+reformatting entire documents in house styles with only a few keystrokes.
 
 % keeping it simple
 To create a {\LaTeX} manuscript,
-you will typically create a new top-level directory for each final output.
-Many projects go so far as to create documents like the main text
+you will typically start from a new top-level directory for each final output,
+such as a paper or report.
+Many projects go so far as to create the main paper
 and any appendix text as separate projects.
```
Comment (kbjarkefur) [inline chapters/7-publication.tex:95]: I think cut or elaborate. As is, this is confusing to someone new to latex. If we want to keep we should explain or give examples to how that is done.

File: chapters/7-publication.tex [hunk 4]
```diff
@@ -67,38 +66,39 @@ \subsection{Preparing manuscripts as {\LaTeX} documents}
...
-For a single output, one convention is to put the manuscript files
+For a single research output, we recommend putting the manuscript files
 (such as \texttt{main.tex} and \texttt{bibliography.bib})
 in the top-level folder,
-and then have the folders for figures, tables, and other inputs.
-Your analytical scripts should easily be able to produce outputs
-at the location of your choice by adjusting paths in the master script;
-output them here when ready.
+and then have subfolders for figures, tables, and other inputs.
+Your analytical scripts should be set up such that it is easy to output files
```
Comment (kbjarkefur) [inline chapters/7-publication.tex:100]:
```suggestion
Your analysis scripts should be set up such that it is easy to output files
```

File: chapters/7-publication.tex [hunk 5]
```diff
@@ -196,22 +197,19 @@ \subsection{Getting started with {\LaTeX} as a team}
 offer the most flexibility to work with {\LaTeX} in teams.
 
 % cloud-based implementations are a good place to start
-Unfortunately, despite these advantages, setting up {\LaTeX} is not always simple,
+Although ultimately worth it, setting up {\LaTeX} is not always simple,
 particularly if you are new to working with plain text code and file management.
 This is because {\LaTeX} requires that all formatting be done in its special code language,
 and it is not particularly informative when you do something wrong.
```
Comment (kbjarkefur) [inline chapters/7-publication.tex:203]:
```suggestion
and it is always informative when you do something wrong.
```
this is a less negative way to say the same thing

File: chapters/7-publication.tex [hunk 6]
```diff
@@ -196,22 +197,19 @@ \subsection{Getting started with {\LaTeX} as a team}
...
-In order to make it as easy as possible for your team
-to use {\LaTeX} without all members having to invest in new skills,
-we suggest using a cloud-based implementation such as Overleaf\sidenote{\url{
-https://www.overleaf.com/
-}}
-as your first foray into {\LaTeX} writing.
-Some teams even adopt these tools as a permanent solution.
+Cloud-based implementations of {\LaTeX} can make it easier for your team to use
+{\LaTeX} without all members having to invest in new skills, 
+and can be particularly useful for first forays into {\LaTeX} writing.
```
Comment (kbjarkefur) [inline chapters/7-publication.tex:210]: `foray` - this is easy to get from context, but can we simplify language?

File: chapters/7-publication.tex [hunk 7]
```diff
@@ -245,106 +243,85 @@ \subsection{Getting started with {\LaTeX} as a team}
 They vary dramatically in their ability to integrate
 with file systems where you store your code and outputs,
 and so you will need to practice an integrated workflow depending what is available to you.
-Despite this, we believe that with minimal learning and workflow adjustments,
-cloud-based implementations are often the easiest way to allow coauthors to write and edit in {\LaTeX},
-so long as you make sure you are available to troubleshoot minor issues like these.
+Some teams adopt cloud-based tools as a permanent solution,
+though others shift to integrated editing and compiling tools such as {\LaTeX} and Atom.
```
Comment (kbjarkefur) [inline chapters/7-publication.tex:247]:
```suggestion
though our recommendation is to eventually shift to 
local editing and compiling using tools such as TexStudio and code editors like Atom. 
```
Some reasons for these edits:
1. Make our recommendation to be to eventually no use cloud based latex setups
2. LaTeX is not a tool. 
3. And I think the alternative to "cloud-based" is not "integrated" but local.
4. I think TexStudio should be mentioned, and Atom as an example of code editors.

File: chapters/7-publication.tex [hunk 8]
```diff
@@ -245,106 +243,85 @@ \subsection{Getting started with {\LaTeX} as a team}
...
+(this may not be necessary if you are working with secondary data that is published elsewhere in this format).
 If you follow the steps outlined in Chapter 5,
 when you get to the publication stage you will have
 a cleaned data set and supporting documentation ready.
-If you follow the steps outlined in Chapter 5,
-when you get to the publication stage you will have
-a cleaned data set and supporting documentation ready.
-Second, you should separately catalog (as supporting material)
+Second, you should separately catalog 
 the analytical dataset used for the output you are publishing.
```
Comment (kbjarkefur) [inline chapters/7-publication.tex:270]:
```suggestion
the analysis dataset used for the output you are publishing.
```

File: chapters/7-publication.tex [hunk 9]
```diff
@@ -353,50 +330,48 @@ \subsection{De-identifying data for publication}
...
 investigate what other results might be obtained from the same population,
 and test alternative approaches or answer other questions.
 This fosters collaboration and may enable researchers to explore variables and
 questions that you do not have time to focus on otherwise.
-There are different options for public data release.
+
+% platforms for publishing data
+The first step toward data publication is choosing the platform where you will publish your data.
+A variety of options exist; 
+it is important to choose one that allows you to obtain a digital object identifier (DOI)
```
Comment (kbjarkefur) [inline chapters/7-publication.tex:371]: Add definition of DOI. This is the first time we mention it. What is different from this and a URL?

File: chapters/7-publication.tex [hunk 10]
```diff
@@ -508,75 +475,74 @@ \subsection{Organizing code for reproducibility}
...
 
 % ensuring code reproducibility
-Before releasing the code, make sure it functions identically on a fresh install of your chosen software.
-A new user should have no problem getting the code to execute perfectly.
-In either a scripts folder or in the root directory,
-you should include a master script that allows the reviewer to run the entire project
-and re-create all raw outputsby changing only a single line of code:
-the one setting the directory path.
+Before publicly releasing a reproducibility package,
+it is essential to make sure that the code executes perfectly on a fresh install of your chosen software.
```
Comment (kbjarkefur) [inline chapters/7-publication.tex:514]:
```suggestion
it is essential to make sure that the code runs identically 
on your individual setup compared to 
a fresh installation of your software.

```
I do not think `perfectly` is the right word. This is the most explicit way to say what we mean

**Comment by kbjarkefur** [comment]
- [ ] Very focused on academic publications, but this should include dashboards, infographics etc and other types of research products useful in the context of development.
- [ ] page 108 -pandoc example - where does this example come from? who has quality controlled that this is the best way to do this?
- [ ] page 108 - "This makes it possible to set up a dedicated GitHub repository for each output and manage contributions and version histories	and manage contributions and version histories using the same system we recommend for analytical work.	using the same system we recommend for analytical work." - sounds like we are recommending setting up a repo per output. Do we want to do that?
- [ ] page 109 - Internet or internet? I know Internet is technically most correct but I am always a bit annoyed on people that capitalize Internet.
- [ ] page 112 - Options for public release. Mention and link to WB options only, and then link to wiki page with more options?

File: bibliography.bib [hunk 11]
```diff
@@ -833,3 +833,12 @@ @MISC{pkg-geometry
...
 }
+
+@Unpublished{Sweeney2000,
+  author  = {Sweeney, Latanya},
+  title   = {Simple Demographics Often Identify People Uniquely},
+  year    = {2000},
+  journal = {Carnegie Mellon University, Data Privacy},
+  type    = {Working paper},
+  url     = {http://dataprivacylab.org/projects/identifiability/},
+}
```
Comment (kbjarkefur) [inline bibliography.bib:844]:
```suggestion

}
```

File: chapters/7-publication.tex (hunk 9 above)
Comment (mariaruth) [reply chapters/7-publication.tex:380]: add sidenote @bbdaniels

Comment (kbjarkefur) [reply chapters/7-publication.tex:380]: @bbdaniels

File: chapters/7-publication.tex (hunk 1 above)
Comment (luizaandrade) [reply chapters/7-publication.tex:624]: @bbdaniels or @mariaruth to add an example

Comment (bbdaniels) [reply chapters/7-publication.tex]: Removed in be8cabb68b76e0b552815f6f60f7a317c9b993fd

File: chapters/7-publication.tex (hunk 9 above)
Comment (bbdaniels) [reply chapters/7-publication.tex:382]: Done in f6eff4990e615c745f54a1e1ae6b42ff1186ec85

Comment (bbdaniels) [reply chapters/7-publication.tex:382]: and d8ffae355a31439c96c91142ff7f0a69e49e500d

File: chapters/7-publication.tex (hunk 7 above)
Comment (bbdaniels) [reply chapters/7-publication.tex]: Done in ab95abe3993028c53482b897e795d04407f49531

**Pull request merged**